simulator/scheduler/plugin/plugins.go

```diff
@@ -208,6 +208,23 @@ func ConvertForSimulator(pls *v1beta2.Plugins) (*v1beta2.Plugins, error) {
 		},
 	}
 
+	defaultPostFilterPls, err := config.InTreePostFilterPluginSet()
```

Comment:
ConvertForSimulator seems repeat the similar logic for each extension point. Can you refactor it to make them done in one function?